python_ta/debug/id_tracker.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Optional
+
+
+class IDTracker:
+    def __init__(self) -> None:
+        self.id_counter: int = 0
+        self.tracker: dict[int, int] = {}
+        self.current_snapshot_objects: set[int] = set()
+
+    def set_id(self, obj) -> Optional[int]:
```

Comment:
The return type is `None`. Also, make sure to include type annotations for all parameters here and throughout.